lib/BackgroundJob/UserImportJob.php

```diff
@@ -59,19 +65,29 @@ public function __construct(
 		$this->logger = $logger;
 		$this->notificationManager = $notificationManager;
 		$this->mapper = $mapper;
+		$this->config = $config;
+		$this->root = $root;
 	}
 
 	public function run($argument): void {
 		$id = $argument['id'];
 
 		$import = $this->mapper->getById($id);
-		$user = $import->getSourceUser();
+		$sourceUser = $import->getSourceUser();
+		$targetUser = $import->getTargetUser();
 		$path = $import->getPath();
+		$dataDir = $this->config->getSystemValueString('datadirectory', \OC::$SERVERROOT . '/data');
```

Comment:
we can't use this, it won't work with primary object store.
never access the filesystem directly, always go through the APIs